distributed/stealing.py

```diff
@@ -199,8 +199,7 @@ def steal_time_ratio(self, ts):
 
         ws = ts.processing_on
         compute_time = ws.processing[ts]
-        if compute_time < 0.005:  # 5ms, just give up
-            return None, None
+        compute_time = max(compute_time, 0.010)
```

Comment:
I've been down this road before, see https://github.com/dask/distributed/pull/4920

I also noticed the way we measure bandwidths is very skewed which impacts this significantly, see https://github.com/dask/distributed/issues/4962#issuecomment-867821099 Not sure if this is still relevant but it was big back then

I still think https://github.com/dask/distributed/pull/4920 is worth getting in but I had to drop this back then because deadlocks popped up.